docs/index.md

```diff
@@ -34,12 +34,34 @@ about pipeline inputs.
 
 ```JSON
 {
-  "QC.read1": "Path to read1",
-  "QC.read2": "Path to read2",
-  "QC.outputDir":"Where the results should be output to",
-  "QC.alwaysRunAdapterClipping": "Boolean (Optional) Whether adapter clipping should always run. Use this if you want to add custom paramaters for read preprocessing. Defaults to 'false'",
+  "QC.read1": "Path to read1"
 }
 ```
+`QC.read1`  is the only required input. In case of read pairs the reverse
+read can be set with `QC.read2`. The adapters for cutadapt can be  set
+with `QC.Cutadapt.adapter` and `QC.Cutadapt.adapterRead2` for read1 and
+read2 respectively. If read1 and read2 use the same adapter this can be
+set with `QC.Cutadapt.adapterBoth`. 
+
+An 
+output directory can be set using an `options.json` file. See [the 
+cromwell documentation](
+https://cromwell.readthedocs.io/en/stable/wf_options/Overview/) for more 
+information. 
+
+Example `options.json` file:
+```JSON
+{
+"final_workflow_outputs_dir": "my-analysis-output",
+"use_relative_output_paths": true,
+"default_runtime_attributes": {
+  "docker_user": "$EUID"
+  }
+}
+```
+Alternatively an output directory can be set with `QC.outputDir`. 
+`QC.outputDir` must be mounted in the docker container. Cromwell should
+be configured correctly to allow this.
```

Comment:
Maybe "Cromwell will need a custom configuration to allow this."

Reply:
:+1: